app/backend/computermanager.h

```diff
@@ -72,6 +74,7 @@ class ComputerManager : public QObject
     void pairHost(NvComputer* computer, QString pin);
 
     void quitRunningApp(NvComputer* computer);
+    Q_INVOKABLE void quitRunningApp(Session* session);
```

Comment:
Why can't we use  `void quitRunningApp(NvComputer* computer)` instead?

Reply:
That would require for NvComputer to inherit from QObject, otherwise it cannot be called from QML. Should that be done instead?